pkg/product/strings.go

```diff
@@ -12,7 +12,7 @@ func LoginCmd() string {
 	case licenseapi.DevPodPro:
 		return "devpod login"
 	case licenseapi.VClusterPro:
-		return "vcluster login"
+		return "vcluster platform login"
```

Comment:
This repository is automatically synced from the [loft-enterprise](https://github.com/loft-sh/loft-enterprise) repository, so this change would need to be made [here](https://github.com/loft-sh/loft-enterprise/blob/main/staging/src/github.com/loft-sh/api/v4/pkg/product/strings.go#L15), otherwise it may be overwritten during the next sync.

Reply:
Hmmm... Got it! Thanks!